crates/apollo-mcp-server/src/introspection/tools/execute.rs

```diff
@@ -58,11 +58,17 @@ impl graphql::Executable for Execute {
             operation_defs(&input.query, self.mutation_mode == MutationMode::All, None)
                 .map_err(|e| McpError::new(ErrorCode::INVALID_PARAMS, e.to_string(), None))?;
 
+        let (_, operation_def, source_path) = operation_defs.ok_or_else(|| {
```

Comment:
Maybe just combine this chain?
```suggestion
                .map_err(|e| McpError::new(ErrorCode::INVALID_PARAMS, e.to_string(), None))?
                .ok_or_else(|| {
```

Reply:
Thanks, @Jephuff! The code looks much cleaner!